utils/install-basic-software.ps1

```diff
@@ -32,16 +32,25 @@ $packages = @(
 echo "Setting up Chocolatey software package manager"
 Get-PackageProvider -Name chocolatey -Force
 
+echo "Setting up Full Chocolatey Install"
+Install-Package -Name Chocolatey -Force -ProviderName chocolatey
+$chocopath = (Get-Package chocolatey | ?{$_.Name -eq "chocolatey"} | Select @{N="Source";E={((($a=($_.Source -split "\\"))[0..($a.length - 2)]) -join "\"),"Tools\chocolateyInstall" -join "\"}} | Select -ExpandProperty Source)
+& $chocopath "upgrade all -y"
+choco uninstall mm-choco.extension --all-versions
```

Comment:
What excatly is `mm-choco.extension`, I went through the script and this line fails since it is not even installed.